src/main/python/linuxband/midi/midi_player.py

```diff
@@ -71,17 +76,17 @@ def startup(self):
         self.__pipew = pipew
 
         pipeName = '/proc/' + str(Glob.PID) + '/fd/' + str(pipew)
-        command = [ Glob.PLAYER_PROGRAM, '-s', '-n', '-x', pipeName ]
+        command = [Glob.PLAYER_PROGRAM, '-s', '-n', '-x', pipeName]
         if Glob.CONSOLE_LOG_LEVEL == logging.DEBUG:
             command.insert(1, '-d')
         try:
             self.__player = subprocess.Popen(command, stdin=subprocess.PIPE)
-        except:
+        except subprocess.CalledProcessError:
```

Comment:
In addition to subprocess.CalledProcessError, subprocess.Popen(...) can throw other types of exceptions. To be on the safe side, I suggest reverting this change.